tools/generate-helpers/rocksdb-bindings/main.go

```diff
@@ -246,6 +266,23 @@ func main() {
 		if err := os.WriteFile("store.go", buf.Bytes(), 0644); err != nil {
 			return err
 		}
+		if props.MigrationSeq == 0 {
+			return nil
+		}
+		buf.Truncate(0)
```

Comment:
Why do we need to truncate? If we truncate do we need to reallocate below?

Reply:
fixed